### ISSUE TITLE
Tomcat rejects square brakets in URL, but they are used by OpenDAP connector libraries

### ISSUE THREAD

**Issue by ashleysommer** [Unidata/tomcat-docker#99]
Using the latest version of `unidata/tomcat-docker`, along with the latest `unidata/thredds-docker`.

Using the NetCDF4 library for Python, connecting to the Thredds Opendap endpoint for our netcdf file. Executing any query on the data using the python library was giving the error:
```
Access Failure
```

Tomcat logs showed it rejected the query for an invalid URL:
```
400 – Bad Request. java.lang.IllegalArgumentException: Invalid character found in the request target. The valid characters are defined in RFC 7230 and RFC 3986.
```

Adding `relaxedQueryChars` to the Connector node in the server.xml in tomcat fixed it:
```
    <Connector server="Apache" secure="true" port="8080" protocol="HTTP/1.1"
            connectionTimeout="20000"
            redirectPort="8443"
            relaxedQueryChars="[,]" />
```

Is it possible to have this small changed added to this docker image?

**Comment by ashleysommer** [comment]
Sorry I just saw this was fixed 9 months ago in here:
https://github.com/Unidata/tomcat-docker/commit/ce8774c0e61aab7d5f6721ddd52ef91ccd1096b4

I wonder why my image is not updating to the latest when I do `docker pull`. I just assumed I'm on the latest.

**Comment by ashleysommer** [comment]
For anyone else coming here while searching for this... I'll explain what I think happened.

Back in late Jan or early Feb 2020 I installed `unidata/thredds-docker:latest` it is based on `unidata/tomcat-docker:8.5` which _should_ be latest. But it didn't have this fix. So for some reason back around that time it seems the latest versions of `thredds-docker` were built on an out of date `tomcat-docker` then manually pushed to dockerhub. This issue has since been resolved (all builds are now done by dockerhub, which ensures dependencies are up to date), so best way for anyone else to fix it is to just `docker pull unidata/thredds-docker` and it will suck in the latest `unidata/tomcat-docker`.
